src/sekoiaio.scss

```
@@ -322,3 +326,44 @@
         display: none;
     }
 }
+
+.alt-mono-font {
+    font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,Courier,monospace;
+}
+
+.changelog .row {
+    display: flex;
+    padding: 24px 64px;
+    border-left: 1px solid #d1d5da;
+    position: relative;
+    padding: 24px 20px 24px 58px;
+}
+.changelog .row::before {
+    transform: translateY(-50%);
+    background: #d1d5da;
+    content: "";
+    left: 0;
+    height: 1px;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 32px;
+    z-index: 1;
+}
+.changelog .row:last-child::after{
+    background: #fff;
+    content: "";
+    height: 50%;
+    position: absolute;
+    left: -1px;
+    bottom: 0;
+    width: 3px;
+    z-index: 0;
+}
+
+.changelog .column-left {
+  flex: 15%;
+}
+.changelog .column-right {
+  flex: 85%;
+}
```

Just SCSS refactoring

```suggestion
.changelog {
    .row {
        display: flex;
        padding: 24px 64px;
        border-left: 1px solid #d1d5da;
        position: relative;
        padding: 24px 20px 24px 58px;

        &::before {
            transform: translateY(-50%);
            background: #d1d5da;
            content: "";
            left: 0;
            height: 1px;
            position: absolute;
            top: 50%;
            transform: translateY(-50%);
            width: 32px;
            z-index: 1;
        }

        &:last-child::after{
            background: #fff;
            content: "";
            height: 50%;
            position: absolute;
            left: -1px;
            bottom: 0;
            width: 3px;
            z-index: 0;
        }
    }

    .column-left {
        flex: 15%;
    }

    .column-right {
        flex: 85%;
    }
}
```